bindings/time-profiler.cc

```diff
@@ -67,12 +67,31 @@ NAN_METHOD(StartProfiling) {
 NAN_METHOD(StopProfiling) {
   Local<String> name = info[0].As<String>();
   CpuProfile* profile =
-    info.GetIsolate()->GetCpuProfiler()->StopProfiling(name);
+      info.GetIsolate()->GetCpuProfiler()->StopProfiling(name);
   Local<Value> translated_profile = TranslateTimeProfile(profile);
   profile->Delete();
   info.GetReturnValue().Set(translated_profile);
 }
 
+void free_buffer_callback(char* data, void* buf) {
+  delete reinterpret_cast<std::vector<char>*>(buf);
+}
+
+NAN_METHOD(StopProfilingProto) {
```

Comment:
Can this be just called StopProfiling? The old StopProfiling and the Translate* methods can be removed I think? The whole protobuf dependency can also go away from the JS code I think?

Reply:
Done.